### ISSUE TITLE
Emby notifications not working

### ISSUE THREAD

**Issue by johnvick** [pymedusa/Medusa#4576]
Branch: master
Commit: 7cfc79d5280b719c6288090fb805521dfc0402cd
Version: v0.2.6
Database: 44.11
Python Version: | 2.7.9 (default, Aug 13 2016, 17:56:53) [GCC 4.9.2]
SSL Version: | OpenSSL 1.0.1t 3 May 2016
OS: | Linux-4.9.61+-armv7l-with-debian-8.11
Locale: | en_GB.UTF-8

I haven't changed the IP, port or API of the Emby device but notifications no longer work. 

This is in the log:
```
2018-07-05 22:43:11 WARNING  Thread_34 :: [7cfc79d] EMBY: Warning: Unable to contact Emby at http://**********/emby/Notifications/Admin: 500 Server Error: Internal Server Error for url: http://**********/emby/Notifications/Admin
```

I think this has just started with the most recent update of Medusa. There haven't been any Emby updates for a couple of weeks. Restarting Emby - no difference. 

Restated Medusa and it updated to:

Branch: master
Commit: e7c4ff3bc9a48b47c932b0e40a539d62e3a62f64
Version: v0.2.6
Database: 44.11
Python Version: | 2.7.9 (default, Aug 13 2016, 17:56:53) [GCC 4.9.2]
SSL Version: | OpenSSL 1.0.1t 3 May 2016
OS: | Linux-4.9.61+-armv7l-with-debian-8.11
Locale: | en_GB.UTF-8

But the problem persists.

Thanks for checking out.

**Comment by sharkykh** [comment]
@johnvick 
Did you change the `Admin` username on Emby?

**Comment by johnvick** [comment]
No, haven't changed any Emby settings (other than changing the API but this was after the problem arose - but this didn't help).

**Comment by sharkykh** [comment]
I need to look into this more, because I don't have Emby installed to check that.

In any case, that's just the notification test. It **should** still work for updating the library from Medusa.
If all the other settings for Emby are correct, of course.

**Comment by johnvick** [comment]
It doesn't trigger the Emby library update as it did before when you click the Emby update button on the TV series screen. Emby is otherwise working fine  as is the Emby/Kodi plugin setup over multiple devices around the house.

**Comment by sharkykh** [comment]
Please check the Emby logs. I can't reproduce this error.

**Comment by johnvick** [comment]
I've tested this again after rebooting every relevant device - I can confirm that the library updates are now working as before, but the test notification still isn't.  Probably not a problem with Medusa - thanks for looking into this and sorry if I have wasted your time.

**Comment by sharkykh** [comment]
The log from Emby did help me figure out what I did wrong with the test notification, so it wasn't all for nothing. Thanks :)

**Comment by sharkykh** [comment]
Fixed with #4622.
